### PR TITLE
Migration

### LINKED ISSUE #139: Migration - DB into NextJS
The DB is currently split out from the Client, we have found it would ease FE and BE communication if we brought it in alongside the Client allowing NextJS to interact directly with the DB.

### PR DESCRIPTION
**Pull request by ZumDeWald**
This PR will initiate moving the DB alongside the Client so that NextJS can directly interact with it. Once this initial step is in place we can begin pulling in information from the DB directly with Next.

Resolves #139

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,7 +46,6 @@ bower_components
 build/Release
 
 # Dependency directories
-node_modules/
```
Comment (Soreg) [inline .gitignore:49]: Isn't it dangerous to remove this? .. We should never commit node_modules :)

Comment (Soreg) [reply .gitignore:49]: Comment / issue resolved - duplicate node_modules was removed

**Review by Soreg — APPROVED** [review]
Looks good :)

**Review by silentkdev — APPROVED** [review]
Sorry for the delay, thanks again!

**Comment by Soreg** [comment]
New pushed changes are tested & approved from my end (Windows).
@ZumDeWald You can merge if your end is OK :)

**Pull request merged**